css/style.css

```diff
@@ -0,0 +1,302 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  
+}
+
+h1 {
+  font-size: 32px;
+}
+
+h2 {
+  font-size: 28px;
+
+
+}
+
+h3 {
+  font-size: 25px;
+}
+
+h5 {
+  font-size: 16px;
+}
+
+p {
+  font-size: 16px;
+  font-weight: 100;
+}
+
+.btn {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  border: none;
+}
+
+.section__background--one {
+  background: radial-gradient(rgb(194, 74, 126), rgb(105, 86, 235));
+}
+
+.contaniner {
+  max-width: 1150px;
+  margin: 0 auto;
+}
+
+.container--max {
+  margin: 0 auto;
+}
+
+.header {
+  padding-top: 2vh;
+}
+
+.nav {
+  display: flex;
+  align-items: space-between;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  color: white;
+}
+
+.nav__link__a {
+  display: block;
+  padding: 10px;
+  text-decoration: none;
+  color: white;
+}
+
+.hero {
+  display: flex;
+  align-items: center;
+  padding-top: 10vh;
+  padding-bottom: 10vh;
+}
+
+.hero__text,
+.hero__image {
+  width: 50%;
+  color: white;
+}
+
+.hero__h1 {
```

Comment:
Staramy się używać nazw własnych, a nie elementów. Bo jak zmienimy wykorzystywany znacznik to wtedy nazwa będzie myląca i będzie trzeba też zmieniać w CSS.